client/components/common/tce-core/AddElement/SelectElement.vue

```diff
@@ -59,17 +60,25 @@ export default {
       const result = filter(this.registry, it => it.type !== 'ASSESSMENT');
       if (this.assessments.length) {
         result.push({
-          name: 'Assessment', type: 'ASSESSMENT', ui: { icon: 'mdi-help' }
+          name: 'Assessment',
+          type: 'ASSESSMENT',
+          ui: { icon: 'mdi-help' }
+        }, {
+          name: 'Reflection',
+          type: 'REFLECTION',
+          ui: { icon: 'mdi-comment-question-outline' }
         });
       }
       if (!this.include) return result;
       return filter(result, it => includes(this.include, it.type));
     },
     assessments() {
-      return filter(this.registry, { type: 'ASSESSMENT' });
+      const types = ['QUESTION'];
```

Comment:
If the goal was to avoid 2 computed for list of questions depending on which type was selected then computed name should be questions.